content/beginner/191_secrets/kms-custom-keystore.md

```diff
@@ -46,18 +44,18 @@ Keep in mind that the KMS Custom Key Store functionality makes use of a minimum
 
 #### Cost
 
-Aside from compliance considerations, your team will want to consider the cost of using this feature. For comparison, I will list the cost of using a CMK created with the default KMS functionality. Then, I will list of the cost of using a CMK created with the custom key store functionality.
+Aside from compliance and security requirements, your may want to consider the cost of using custom key stores. Below you can find a cost comparison between default AWS KMS key store and AWS KMS custom key store.
```

Comment:
"your may" -> "you may"

Also add a note that this assumes a certain region and provide a link where they can find the latest KMS pricing information (to https://aws.amazon.com/kms/pricing/).